api/envoy/config/route/v3/route_components.proto

```diff
@@ -300,6 +300,11 @@ message Route {
   // If set, the bytes actually buffered will be the minimum value of this and the
   // listener per_connection_buffer_limit_bytes.
   google.protobuf.UInt32Value per_request_buffer_limit_bytes = 16;
+
+  // [#not-implemented-hide:]
+  // Additional clusters which are not referenced directly by a RouteAction
+  // cluster_specifier, but may be selected by a cluster_specifier_plugin.
+  repeated string additional_clusters = 19;
```

Comment:
I think this field should be in `RouteConfiguration`, not in `Route`.  That way, if there are many routes that use the same `cluster_specifier_plugin`, we don't need to specify this list of additional clusters multiple times.